src/block/blog-posts/style.js

```diff
@@ -63,6 +64,50 @@ export const createStyles = props => {
 		} )
 	}
 
+	// Container.
+	const {
+		columnPaddingUnit = 'px',
+		tabletColumnPaddingUnit = 'px',
+		mobileColumnPaddingUnit = 'px',
+	} = props.attributes
+
+	let columnSelector
+
+	if ( [ 'vertical-card', 'horizontal-card' ].includes( design ) ) {
+		columnSelector = '.ugb-blog-posts__content'
+	} else if ( design === 'image-card' ) {
+		columnSelector = '.ugb-blog-posts__header, .ugb-blog-posts__content'
+	} else {
+		columnSelector = '> .ugb-inner-block > .ugb-block-content > *'
+	}
```

Comment:
Please move this to Premium, you can use a filter here to allow the premium version to change the selector:

```js
const columnSelector = applyFilters( 'stackable.blog-posts.spacing-paddings.selector', '> .ugb-inner-block > .ugb-block-content > *', props )
```